GearRent/Models/Listing.m

```diff
@@ -19,11 +19,14 @@ @implementation Item
 @dynamic ownerId;
 @dynamic tags;
 @dynamic geoPoint;
-@dynamic city;
+@dynamic location;
 @dynamic reservations;
 @dynamic availabilities;
 @dynamic isAlwaysAvailable;
 @dynamic geohash;
+@dynamic dynamicPrice;
+@dynamic minPrice;
+@dynamic categoryId;
```

Comment:
do we need to declare all the properties dynamic here?

Reply:
from my knowledge, yes. I asked julian last week about it and he said it tells objective c that these properties will be decided by the accessors ( in this case within the program )

Reply:
It tells the compiler those properties will be generated dynamically during runtime (https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ObjCRuntimeGuide/Articles/ocrtDynamicResolution.html#//apple_ref/doc/uid/TP40008048-CH102-SW1).